js/download.js

```diff
@@ -11,3 +11,108 @@ function showHideFun() {
     secondary_btns.style.display = "block";
   } else secondary_btns.style.display = "none";
 }
+
+// set the canvas size according to the device screen size
+let width = "1440";
+
+if (window.screen.width >= 1024) {
+  width = "1920";
+} else if (window.screen.width >= 768) {
+  width = "2048";
+}
+
+let height = Math.round((width * window.screen.height) / window.screen.width);
+
+// get the img URL from the localStorage
+
+var img = new Image();
+img.src = localStorage.getItem("BG");
+img.crossOrigin = "Anonymous";
+
+// create canvas with the image as a background and the quote from the localStorage on it
+
+var canvas = document.getElementById("canvas");
+canvas.setAttribute("width", width);
+canvas.setAttribute("height", height);
+var ctx = canvas.getContext("2d");
+img.onload = drawImageAndQuote.bind(null, img, ctx);
```

Comment:
The onload attribute takes a function (callbackFn) but
There is no need to use bind method as you have not used "this" keyword in your function
Try using a function instead of like this
`img.onload = ()=>drawImageAndQuote(img, ctx);`